### ISSUE TITLE
Registry API Service doesn't build anymore

### ISSUE THREAD

**Issue by nutjob4life** [NASA-PDS/registry-api-service#91]
## 🐛 Describe the bug

The current `main` of `registry-api-service` fails to build due to a bad dependency:

```console
$ cd /tmp
$ git clone git@github.com:NASA-PDS/registry-api-service.git
…
$ cd registry-api-service
$ mvn clean install
…
[ERROR] Failed to execute goal on project registry-api-service: Could not resolve dependencies for project gov.nasa.pds:registry-api-service:jar:0.5.0-SNAPSHOT: Failure to find gov.nasa.pds:api:jar:pds.api.110 in https://oss.sonatype.org/content/repositories/snapshots was cached in the local repository, resolution will not be reattempted until the update interval of oss.sonatype.org-snapshot has elapsed or updates are forced -> [Help 1]
…
$ echo \U+1F616
😖
```

It looks like in 7a9456a034d4c23d4b6817522d7bab5e8e3d48a2 a typo was introduced in the dependency for `gov.nasa.ods:api` changing its version from `0.5.0-SNAPSHOT` to an unusual version tag `pds.api.110`. Is this correct?

Hat tip to @ramesh-maddegoda for discovering this! 📣

## 🕵️ Expected behavior

```console
$ mvn clean install
…
BUILD SUCCESS
…
```

**Comment by jordanpadams** [comment]
@nutjob4life I think this is related to the issues we have been having with our versioning / branching approach for PDS API development. adding as part of parent epic https://github.com/NASA-PDS/pds-api/issues/114. hopefully this will be addressed soon. is this a blocker for you or @ramesh-maddegoda ?

**Comment by ramesh-maddegoda** [comment]
> @nutjob4life I think this is related to the issues we have been having with our versioning / branching approach for PDS API development. adding as part of parent epic [NASA-PDS/pds-api#114](https://github.com/NASA-PDS/pds-api/issues/114). hopefully this will be addressed soon. is this a blocker for you or @ramesh-maddegoda ?

@jordanpadams This is not a blocker for me. I can continue with rest of the components for now.

**Comment by tloubrieu-jpl** [comment]
Hi @ramesh-maddegoda , @nutjob4life , @jordanpadams 
I made the fix in commit https://github.com/NASA-PDS/registry-api-service/commit/b4547bdae03dfa29c7dd3662eca2e4c79e0acbed

There error was due do a 'bad' merge, I missed to update the 'feature version' dependency that @al-niessner used purposefully for his development before I merge into main.

As @jordanpadams said this error in trhe background is aimed at being resolved by  NASA-PDS/pds-api#114 . The soution for that is going to merge registry-api-service and pds-api-client into a single repository, see https://docs.google.com/spreadsheets/d/1jTP_21jIW9wLnncdAKZgeIESf5eI4HSHQCAe3mVVG-E/edit#gid=0
